### PR TITLE
[12.0] [MIG] account_invoice_fiscal_position_update

### PR DESCRIPTION
**Pull request by bjeficent**
Bump to version 12.0

### CODE REVIEW

File: account_invoice_fiscal_position_update/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# © 2011-2014 Julius Network Solutions SARL <contact@julius.fr>
+# © 2014-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# © 2018 Roel Adriaans <roel@road-support.nl>
```
Comment (cubells) [inline account_invoice_fiscal_position_update/__manifest__.py]: change `©` by `Copyright` please.

File: account_invoice_fiscal_position_update/models/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# © 2011-2014 Julius Network Solutions SARL <contact@julius.fr>
+# © 2014 Akretion (http://www.akretion.com)
```
Comment (cubells) [inline account_invoice_fiscal_position_update/models/account_invoice.py]: Change `©` by `Copyright` please.

File: account_invoice_fiscal_position_update/models/account_invoice.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+# @author Mathieu Vatel <mathieu _at_ julius.fr>
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models, api, _
```
Comment (cubells) [inline account_invoice_fiscal_position_update/models/account_invoice.py]: change order please: `from odoo import _, api, moddels`

File: account_invoice_fiscal_position_update/tests/test_inv_fiscal_pos_update.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
+# © 2014 ToDay Akretion (http://www.akretion.com)
```
Comment (cubells) [inline account_invoice_fiscal_position_update/tests/test_inv_fiscal_pos_update.py]: Change `©` by `Copyright` please.

**Review by cubells — CHANGES_REQUESTED** [review]
Minor changes. Ready to approve.

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing.

**Comment by legalsylvain** [comment]
Hi @bjeficent. Thanks for porting this module.
~~Just a question, I don't understand why this module recompute account (income / expense) and write it on each line. AFAIK, the income / expense account doens't depends on fiscal positions. FP are just about taxes.~~
~~I think it should be better to separate the two features~~
~~- update taxes (account_invoice_fiscal_position_update)~~
~~- Reset accounts (account_invoice_account_reset)~~

~~Don't you think ?~~

~~Thanks for the explanation.~~

OK, I just checked that fiscal position also allow to map account. I didn't know that base feature. Sorry for the noice. :+1:

**Review by legalsylvain — APPROVED** [review]

**Comment by alexis-via** [comment]
The filter of lines on display_type for forgotten in this PR. The display_type field on invoice lines was introduced in v12.

**Pull request merged**